egs/babel_multilang/s5/local/nnet3/run_tdnn_multilingual.sh

```diff
@@ -0,0 +1,296 @@
+                    # ivector_suffix = ''.
+
+bnf_dim=           # If non-empty, the bottleneck layer with this dimension is added at two layers before softmax.
+dir=exp/nnet3/multi_bnf
+
+. ./path.sh
+. ./cmd.sh
+. ./utils/parse_options.sh
+
+[ ! -f local.conf ] && echo 'the file local.conf does not exist!' && exit 1;
+. local.conf || exit 1;
+
+
+num_langs=${#lang_list[@]}
+
+echo "$0 $@"  # Print the command line for logging
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+for lang_index in `seq 0 $[$num_langs-1]`; do
+  for f in data/${lang_list[$lang_index]}/train/{feats.scp,text} exp/${lang_list[$lang_index]}/$alidir/ali.1.gz exp/${lang_list[$lang_index]}/$alidir/tree; do
+    [ ! -f $f ] && echo "$0: no such file $f" && exit 1;
+  done
+done
+
+if [ "$speed_perturb" == "true" ]; then suffix=_sp; fi
+dir=${dir}${suffix}
+
+if $use_pitch; then feat_suffix=${feat_suffix}_pitch ; fi
+
+for lang_index in `seq 0 $[$num_langs-1]`; do
+  echo "$0: extract high resolution 40dim MFCC + pitch for speed-perturbed data "
+  echo "and extract alignment."
+  local/nnet3/run_common_langs.sh --stage $stage \
+    --use-pitch $use_pitch \
+    --speed-perturb $speed_perturb ${lang_list[$lang_index]} || exit 1;
+done
+
+if $use_ivector; then
+  ivector_suffix=
+  if [ -z "$ivector_extractor" ]; then
+    mkdir -p data/multi
+    mkdir -p exp/multi/nnet3
+    global_extractor=exp/multi/nnet3
+    ivector_extractor=$global_extractor/extractor
+    multi_dir_data=data/multi/train${suffix}_hires
+    ivector_suffix=_gb
+    echo "$0: combine training data using all langs for training global i-vector extractor."
+    if [ ! -f $multi_dir_data/.done ]; then
+      echo ---------------------------------------------------------------------
+      echo "Pooling training data in $multi_dir_data on" `date`
+      echo ---------------------------------------------------------------------
+      mkdir -p $multi_dir_data
+      combine_lang_list=""
+      for lang_index in `seq 0 $[$num_langs-1]`;do
+        combine_lang_list="$combine_lang_list data/${lang_list[$lang_index]}/train${suffix}_hires"
+      done
+      utils/combine_data.sh data/multi/train${suffix}_hires $combine_lang_list
+      utils/validate_data_dir.sh --no-feats data/multi/train${suffix}_hires
+      touch data/multi/train${suffix}_hires/.done
+    fi
+  fi
+  if [ ! -f $global_extractor/extractor/.done ]; then
+    echo "$0: Generate global i-vector extractor on pooled data from all "
+    echo "languages in $multi_data_dir, using an LDA+MLLT transform trained "
+    echo "on ${lang_list[0]}."
+    local/nnet3/run_shared_ivector_extractor.sh  \
+      --suffix $suffix \
+      --stage $stage ${lang_list[0]} \
+      $multi_data_dir $global_extractor || exit 1;
```

Comment:
I think there is a parameter missing --something like data/multi/train${suffix}_hires or something else...